config.js

```diff
@@ -476,6 +476,9 @@ var config = {
     // Hides add breakout room button
     // hideAddRoomButton: false,
 
+    // Disables join breakout room for non-moderator participants.
+    // joinRoomModeratorOnly: false,
```

Comment:
This option has no "breakout" in the name, so it'd weird to see out of context. I realize that's also the case with the previous one 😢 

I'd say it's time to add a `breakoutRooms` section where we put the options (remember to add backwards compatibility).

Reply:
No :-) We generally do it as we add the new option, otherwise we need to immediately add backwards compat code, which is not ideal.